### DISCUSSION TITLE
My Profile and Repositories doesn't show on google search

### DISCUSSION THREAD

**Discussion by jpdoshi** [community/community#68416] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

My profile url is https://gihub.com/jpdoshi this account is neither new, nor private. I have provided solid description in profile readme and bio. It shows up on bing, duckduckgo, yahoo, etc search results, but not on google. i tried site:github.com/jpdoshi, it didn't show anything. some suggest google search console, but i don't have any experience of SEO.

**Comment by tuhamworld** [comment, marked as answer]
Hello there, sorry to hear about that.

I would suggest working on more projects for your repository, contributing to other projects on GitHub, including your GitHub link on other popular social media profiles you own, and participating in a coding challenge (e.g. Frontend Mentor).

  I'm hoping following these tips will help your GitHub profile show on Google search results. 

Best regards

**Reply by jpdoshi** [reply]
@tuhamworld first of all thanks for your answer, i will try steps you suggested. hope it works 🤞

**Reply by tuhamworld** [reply]
Glad you're able to find this helpful.
 Best of luck 🚀

**Comment by melvincwng** [comment]
Hey @jpdoshi, it takes time for the web crawlers in Google's search engine to crawl through and index websites in their search engine.

There isn't much that can be done at this point in time, but to patiently wait for Google to index your website (GitHub profile).

Based on experience, your GitHub profile will most likely be searchable on Google in the future - it just takes time.

Hope this clarifies, take care and have a great day ahead 👍🏻  

References: 
1. [Google Search Central Official Documentation](https://developers.google.com/search/docs/crawling-indexing/ask-google-to-recrawl#:~:text=Crawling%20can%20take%20anywhere%20from,instantly%20or%20even%20at%20all.)
2. [Similar issue previously faced by another GitHub Community member](https://github.com/orgs/community/discussions/44421)

**Reply by jpdoshi** [reply]
@melvincwng thanks for your answer, i shall wait some days maybe google might take more time.

**Reply by melvincwng** [reply]
@jpdoshi Awesome 👌

**Comment by taoliujun** [comment]
Hello @jpdoshi ! Have you solve it with @tuhamworld 's answer?

**Reply by jpdoshi** [reply]
yep, updating profile and sharing github profile link on other platforms made it work, such that google think it's relevant content

**Reply by Wayne0758** [reply]
Helpful!

**Comment by im-amal-raj** [comment]
Hi @jpdoshi, I'm also facing the same problem. I recently changed my username and name—now it's https://github.com/im-amal-raj. But when I search site:github.com "im-amal-raj", my friend's profile appears instead; he worked on one of my projects. I created a portfolio website, linked my GitHub there, and added the website to Google Search Console, but my GitHub profile still isn't appearing. Can you tell me the exact changes you made, or anything you found to be quite impactful?

**Reply by jpdoshi** [reply]
I tried updated my profile, included links of my socials. then I also shared my github profile link in socials and my portfolio website: https://jpdoshi.vercel.app then it took 2-3 weeks to list on google search. now I'm not updating/optimizing things so ranking did go down in search results. but now that I think many people are using AI chatbots instead of google search, so I don't think google will be used as much in future.

**Reply by im-amal-raj** [reply]
Thank you for replying so quickly. This thread was created so long ago that I wasn’t even sure you'd notice it. Maybe it could take weeks like you said, so I will wait, I guess. I totally agree with your point about AI replacing normal search.  I’ve noticed that I search less the normal way and use ai more instead. I think a big part of that is how AI does the thinking for us. Instead of coming up with a plan and then hunting for the answer ourselves, we just tell ai something like “find me x,” and it curates the best response based on our chat history and other factors, looking at things from all possible angles. Even google has started implementing that at the top of their website lol. I wonder how we can get ai to pick up our content.

**Comment by JoshLomax** [comment]
<img width="292.5" height="630.75" alt="IMG_1476" src="https://github.com/user-attachments/assets/ce6e181b-3138-40b9-8a5c-5eebe47195c2" />

I pasted a link to my profile across other accounts and saw some improvement but this random url is the highest result for my profile. What is this?

**Comment by yesanqian** [comment]
In fact, about a month ago, I searched for relevant information on this topic.

I've created a GitHub repository for those who want to bypass Telegram channel or group copy-forward restrictions (e.g., restricted saving/downloading of content)(Restrict saving content,Telegram  download restricions):

https://github.com/yesanqian/tg-message-clone

As you can see, it has now been successfully indexed by Google.

**First, I want to sincerely thank everyone who offered advice on this matter!**

Here are the steps I took to help with the indexing:

> 1. I updated my profile and uploaded a new avatar.
> 
> 2. I shared the repository link on other Google-indexed sites (for better visibility and backlinks, which may take time to take effect). 
> 
> 3. I participated in some interactions on GitHub, such as joining discussions.
> 
> 4. I forked someone else's project.
> 
> 5. I waited for about a month.

**Hopefully, these steps will help others who are trying to solve the same problem.**

**Reply by sifaksousmail** [reply]
Hello, may I ask how long it took ?
